NEWS

```diff
@@ -1,3 +1,19 @@
+09-Aug-2018 IGNITION v0.27.0
+
+  Features:
+
+    - Ignition is now built as a Position Independent Executable (PIE)
+    - Blackbox tests now run against all spec versions (within the same major
+      version) greater than their minimum version
+    - Report Ignition status to cloud when running on Packet
```

Comment:
Tense change, Fix it if you like, I don't think it's super important either way.

Reply:
I tweaked it. :arrow_down: